manticore/core/workspace.py

```diff
@@ -129,15 +129,16 @@ def load_value(self, key, binary=False):
             return s.read()
 
     @contextmanager
-    def save_stream(self, key, *rest, **kwargs):
+    def save_stream(self, key, binary=False, *rest, **kwargs):
```

Comment:
Is the `=False` okay? It seems to me that it changes previous behavior in which we assumed it to be `True` if it wasn't passed.

Also, we should probably remove `*rest` and `**kwargs` as they are not used.

Reply:
it does change previous behavior, but i made sure all calls to it are using this new api

and good idea, removing that and the *rest **kwargs from __named_stream too